assets/script.js

```diff
@@ -469,15 +469,16 @@ function fetchCandidateTravels() {
             return response.json();
         })
         .then(function (locRes) {
-            console.log("ProPublica Trips", locRes);
+            // console.log("ProPublica Trips", locRes);
             
             Candidate.travelCard.tripsCandidate = [];
             Candidate.travelCard.tripsOther = [];
 
             if(locRes.num_results > 0) {
                 parseTrips(locRes.results);
             }
```

Comment:
Will,
Could you please put this lines up (into if statement) before }

//calls to generate travel card
            fillTravelCard(Candidate.travelCard.tripsCandidate);
            
            I've just noticed that it should be inside the if because it relies on parsed trip data. Thanks.

Reply:
That is a good optimization. I'll fix this issue and push up the adjustment. Thanks!